packages/ffe-buttons-react/src/BackButton.md

```diff
@@ -1,3 +1,19 @@
 ```js
 <BackButton onClick={f => f}>Tilbake</BackButton>
 ```
+
+Variant _dark_ for interne løsninger med mørk bakgrunn.
+
+```js { "props": { "className": "sb1ds-example-dark" } }
```

Comment:
Is this `{ "props": { "className": "sb1ds-example-dark" } }` stuff some kind of magic way to declare props?

I'm also pretty sure you don't really need `React.Fragment`

Reply:
Never mind, figured out the `props` thing, that's neat.